builder/amazon/common/run_config.go

```diff
@@ -44,6 +44,20 @@ type SecurityGroupFilterOptions struct {
 	config.NameValueFilter `mapstructure:",squash"`
 }
 
+// [Metadata Options](#metadata-settings)
```

Comment:
This link points to the current section. Check it out: https://packer-og00506pb.vercel.app/docs/builders/amazon/ebs#metadata-settings
```suggestion
// Configures the metadata options.
```